wit/handler.wit

```diff
@@ -0,0 +1,8 @@
+/// An interface for a consumer relying on push-based message delivery.
+default interface handler {
+    use pkg.types.{event}
+    use err.tracing.{error}
+
+	/// Creates an on-receive handler push-based message delivery.
```

Comment:
indentation looks a little odd here

Reply:
Similar on other wit files. Is it common to indent a different depth for `use` and func defs?

Reply:
Nop~ not sure how that happened. In any case, fixed ✅